src/patternfly/components/Progress/progress.scss

```diff
@@ -129,20 +124,20 @@
 
   // status modifiers
   &.pf-m-success {
-    --#{$progress}__bar--before--BackgroundColor: var(--#{$progress}--m-success__bar--BackgroundColor);
+    --#{$progress}__indicator--BackgroundColor: var(--#{$progress}--m-success__bar--BackgroundColor);
     --#{$progress}__status-icon--Color: var(--#{$progress}--m-success__status-icon--Color);
-    --#{$progress}--m-inside__measure--Color: var(--#{$progress}--m-success--m-inside__measure--Color);
   }
 
   &.pf-m-warning {
-    --#{$progress}__bar--before--BackgroundColor: var(--#{$progress}--m-warning__bar--BackgroundColor);
+    --#{$progress}__indicator--BackgroundColor: var(--#{$progress}--m-warning__bar--BackgroundColor);
     --#{$progress}__status-icon--Color: var(--#{$progress}--m-warning__status-icon--Color);
     --#{$progress}--m-inside__measure--Color: var(--#{$progress}--m-warning--m-inside__measure--Color);
   }
-
+  
   &.pf-m-danger {
-    --#{$progress}__bar--before--BackgroundColor: var(--#{$progress}--m-danger__bar--BackgroundColor);
+    --#{$progress}__indicator--BackgroundColor: var(--#{$progress}--m-danger__bar--BackgroundColor);
```

Comment:
Just a nit, but since these are changes to the indicator color, the value used here should probably be `var(--#{$progress}--m-danger__indicator--BackgroundColor)`